### ISSUE TITLE
Add tox environment to reformat code

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#608]
Although `tox -e check` tells you what's wrong with the code, it's not obvious for newcomers how to fix those issues. We should have a `tox -e format` or similar that actually reformats the code.

**Comment by azlkiniue** [comment]
I am interested to try this.
From this [comment](https://github.com/poliastro/poliastro/pull/640#discussion_r27183904) I know that we need to use `black` and `isort`. Is it enough?

**Comment by Juanlu001** [comment]
First `isort`, then `black` :) Correct!

**Comment by Juanlu001** [comment]
@azlkiniue still thinking about working on this?

**Comment by azlkiniue** [comment]
> @azlkiniue still thinking about working on this?

Oh, I'm sorry.. almost forget this one..
Are these commands correct?
```
isort --recursive --section-default THIRDPARTY src setup.py
black src setup.py
```

**Comment by Juanlu001** [comment]
The isort one would be `isort --recursive --project poliastro --section-default THIRDPARTY src setup.py`. If you have poliastro installed in development mode it would be just `isort --recursive src setup.py`, but I prefer the safe version.

**Comment by Juanlu001** [comment]
Closed in #655
